tests/unit_test/twitter_ads_api/twitter_ads_api_pipeline_test.py

```diff
@@ -36,33 +34,63 @@
 
 FROM_BIGQUERY_PARAM_VALUE = ['bq_param_1']
 START_TIME_PARAM_VALUE = 'start_time_value_1'
+END_TIME_PARAM_VALUE = 'end_time_value_1'
+PLACEMENT_PARAM_VALUE = ['placement_value_1']
```

Comment:
rename to something like `SINGLE_PLACEMENT_PARAM_VALUE`? (as some tests do rely on that)